### ISSUE TITLE
Large building bug when attacked.

### ISSUE THREAD

**Issue by space-jam-** [snipplets/Dota-2-Building-Helper#8]
Some melee units struggle to attack larger buildings (size 4+). The gridnav blockers seem to wig out the pathfinding, try changing using SetHullSize on buildings to overcome this.

**Comment by int128fly** [comment]
I beleive your the one who have to assign the correct HullSize, also there is a lua command if there is no correct preset hull for your unit.

**Comment by MNoya** [comment]
Yup, I had this same issue today, it's not really BH's fault, just gotta have enough Hull for units to reach the hit
